docs/examples/AccelerometerExample/src/AccelerometerExample.ls

```diff
@@ -59,10 +68,13 @@ package
             sprite.y = Math.clamp(sprite.y, 0, stage.stageHeight);
         }
 
-        protected function onAccelerometerData(x:Number, y:Number, z:Number) 
+        protected function onAccelerometerData(type:MobileSensorType, x:Number, y:Number, z:Number)
         {
+            if (type != MobileSensorType.Accelerometer)
+                return;
+
             directionX = x;
-            directionY = -y; 
+            directionY = -y;
```

Comment:
Alright, if it's consistent with the behavior in the last sprint I think it should be ok.